pom.xml

```diff
@@ -21,9 +21,9 @@
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <java.level>8</java.level>
     <jira-rest-client.version>5.2.2</jira-rest-client.version>
-    <fugue.version>3.0.0</fugue.version>
+    <fugue.version>4.7.2</fugue.version>
     <!-- jenkins -->
-    <jenkins.version>2.277.4</jenkins.version>
+    <jenkins.version>2.324</jenkins.version>
```

Comment:
I definitely prefer wait for next LTS including guava upgrade. it's coming soon.
the plugin is currently working with guava upgrade.
I don't why we should rush the upgrade and prevent a lot users using the new releases of the plugin.